layouts/_default/baseof.html

```diff
@@ -60,15 +60,15 @@
             <nav id="main-nav">
                 <div id="menu-items">
                     <a href="{{ (partial "functions/link" (dict "slug" "/generator")).url }}" class="menu-link desktop-only">{{ T "header-generator" }}</a>
-                    <a href="{{ (partial "functions/link" (dict "slug" "/my-requests")).url }}" class="menu-link desktop-only  my-requests-link">{{ T "header-my-requests" }}</a>
+                    <a href="{{ (partial "functions/link" (dict "slug" "/my-requests")).url }}" class="menu-link desktop-only my-requests-link">{{ T "header-my-requests" }}</a>
                     <a href="{{ (partial "functions/link" (dict "slug" "/blog")).url }}" class="menu-link desktop-only">{{ T "header-blog" }}</a>
```

Comment:
I think we should swap 'Blog' and 'My requests' to match the order on desktop.

Reply:
I wanted to keep the priorities similiar to the desktop, where blog and my requests have their own link, therefore I put them on the top. But I agree, it might be confusing. :D I'll change it.